### ISSUE TITLE
The DEX needs a logout button

### ISSUE THREAD

**Issue by ptyx11** [atomiclabs/hyperdex#214]
---

HyperDEX Nightly 0.0.0-570351d-18.5.13.937
Electron 2.0.0
darwin 17.5.0
Locale: en-US

**Comment by sindresorhus** [comment]
You can log out from the HyperDEX (macOS) / File (Linux/Window) menu.

**Comment by lukechilds** [comment]
@sindresorhus We probably should make this easier to access. Maybe clicking the avatar in the top right corner should open a drop down menu. That seems like the intuitive place to look.

Not vital for the `0.1.0` milestone but we should do this at some point.

**Comment by sindresorhus** [comment]
@lukechilds I don't agree. This is a native app, not a web app. We should use the native menus for things like this.

Here are how some other native apps do it:

<img width="549" alt="screen shot 2018-05-14 at 18 34 24" src="https://user-images.githubusercontent.com/170270/39995554-bf0fa350-57a6-11e8-83d8-8862bd3645db.png">

<img width="374" alt="screen shot 2018-05-14 at 18 40 50" src="https://user-images.githubusercontent.com/170270/39995560-c912b5ea-57a6-11e8-9142-3aebe1e418c6.png">
<img width="537" alt="screen shot 2018-05-14 at 18 41 36" src="https://user-images.githubusercontent.com/170270/39995562-c9893b02-57a6-11e8-95e9-d5e1e7f0f1cf.png">
<img width="626" alt="screen shot 2018-05-14 at 18 42 04" src="https://user-images.githubusercontent.com/170270/39995563-c9bdc606-57a6-11e8-9c38-6a5b4ae737c4.png">

**Comment by ptyx11** [comment]
An easy to use Logout button to use another profile isn't going to kill anyone. Just place it in the corner like most apps. This is about user friendliness.

**Comment by sindresorhus** [comment]
> Just place it in the corner like most apps.

What native apps? I'm just curious as I've never seen it done in the UI in native apps.

**Comment by sindresorhus** [comment]
We could make it more obvious by putting in the Portfolio menu, like this:

<img width="421" alt="screen shot 2018-05-14 at 18 53 53" src="https://user-images.githubusercontent.com/170270/39995974-36dbe1a4-57a8-11e8-9d7f-9394ad1d5d18.png">

**Comment by lukechilds** [comment]
I like adding it to a portfolio menu.

And just to counter your point, I think it's slightly different in our case. In those other apps you mentioned users aren't likely to be switching accounts very often. The user accounts are more for a subscription model so once their logged in it's likely to stay that way, probably forever.

However portfolios are a fundamental part of our app, logging in/out is likely to occur much more often. And if you compare to an app like Spotify, which has a similar kind of UI design to us, and also will be more likely to have multiple users to log in/out, they do have an in app login button in the corner.

<img width="1149" alt="screen shot 2018-05-14 at 6 59 30 pm" src="https://user-images.githubusercontent.com/2123375/39996208-fa738928-57a8-11e8-8741-e347c640d916.png">

**Comment by sindresorhus** [comment]
I'm happy to go with whatever you decide @lukechilds, but I still don't agree it's any easier or more obvious to click the avatar and then "Log Out" than it is to click the "Portfolio" menu and then "Log Out". Exact same amount of clicks and mouse movement and I would even argue the system menu is more obvious as it's text instead of behind an avatar icon.

**Comment by norbertdragan** [comment]
My assumptuon is that in this case slightly more users will intuitively search for the logout at the corner dropdown.

But why not offer this option at both places?
